code/game/objects/items/AI_modules.dm

```diff
@@ -523,73 +527,93 @@ AI MODULES
 	playsound(user, 'sound/machines/click.ogg', 20, TRUE)
 	src.loc.visible_message("<span class='warning'>[icon2html(src, viewers(loc))] [laws[1]]</span>")
 
-/******************** Mother Drone  ******************/
+
+/******************** Mother Drone ******************/
 
 /obj/item/ai_module/core/full/drone
 	name = "'Mother Drone' Core AI Module"
 	law_id = "drone"
+	desc = "An AI Module for changing the programming of an AI's core laws. There seems to be a warning label on the back..."
+
+/obj/item/ai_module/core/full/drone/show_laws(mob/user as mob)
```

Comment:
Why is this in show_laws and not examine? And why do you need `as mob`?

Reply:
as I mentioned before, this is an extension of the show_laws proc so that it will show up AFTER the list of laws if the lawset is examined while close enough to see its list of laws.

Reply:
oh, wait, this will get shown to AIs who press the show laws button... I'll fix that.